index.html

```diff
@@ -4496,72 +4853,32 @@ <h3 id="Warrior">Warrior</h3>
                </td>
                <td><strong>Let out a roar that terrifies those who hear it.</strong>
                   <p>
-                     Inflict <a class="condition" data-tooltip-container>fear</a> to all enemies within 15ft. 1 WILL point to resist the
+                     Inflict <a class="condition" data-tooltip-container>fear</a> to all enemies within 10ft. 1 WILL point to resist the
                      <a class="condition" data-tooltip-container>fear</a>.
                </td>
-               <td>E, 15ft, All within range (AoE)
+               <td>E, 10ft, All within range (AoE)
                </td>
                <td>1
                </td>
             </tr>
             <tr>
                <td>3
                </td>
-               <td>Debilitate
+               <td>Pin and Crush
                </td>
-               <td><strong>Batter your enemy with a crippling and critical strike.</strong>
+               <td><strong>Pin a foe and ready your weapon for a massive attack.</strong>
                   <p>
-                     Make a melee weapon attack against a creature. On a hit the creature is
-                     <a class="condition" data-tooltip-container>immobilised</a> or <a class="condition" data-tooltip-container>stunned</a> (your choice). If the creature already
-                     has the condition you chose, the attack is a critical hit. 2 FORT points to
-                     resist the <a class="condition" data-tooltip-container>immobilise</a> or <a class="condition" data-tooltip-container>stun</a> - the critical hit cannot be
-                     resisted.
+                     Select a creature within 5ft and <a class="condition" data-tooltip-container>immobilise</a> it.
+                     3 FORT or REFL points to resist the <a class="condition" data-tooltip-container>immobilise</a>.
+                     At the start of your next turn, before performing any other actions,
```

Comment:
This is cool, not sure it needs to be a crit.  I fear it's impact vs something which just doesn't move a lot (like a stone golem boss) where you can just chain crits even if the imbolisation doesn't work.  A distributed 2AP for an attack is not bad, plus the sig resistance drain

Reply:
Yeah potentially, I think it does need a bigger payoff than just a regular attack, but a crit may be too much